helm_deploy/values-dev.yaml

```diff
@@ -27,6 +27,7 @@ generic-service:
 
     ACTIVE_AGENCIES: '***'
     REVIEWS_PRISONS_ENABLED: "BXI, WDI"
+    COMPLETED_GOALS_ENABLED: "true"
```

Comment:
I would leave it disabled in `dev` for now ; otherwise we'll end up deploying half baked stuff to the `dev` environment. We can enable it in `dev` once we (both you and Stephen) have finished the dev and we want the team to test it